driver/config/config.go

```diff
@@ -701,8 +702,16 @@ func (p *Config) CourierSMTPFromName() string {
 	return p.p.StringF(ViperKeyCourierSMTPFromName, "")
 }
 
-func (p *Config) CourierTemplatesRoot() string {
-	return p.p.StringF(ViperKeyCourierTemplatesPath, "courier/builtin/templates")
+func (p *Config) CourierTemplatePath(subpath ...string) string {
+	var fullpath []string
+	if p.p.Koanf.Exists(ViperKeyCourierTemplatesPath) {
+		fullpath = []string{p.p.String(ViperKeyCourierTemplatesPath)}
+	} else {
+		fullpath = []string{"courier", "builtin", "templates"}
+	}
+	fullpath = append(fullpath, subpath...)
+
+	return path.Join(fullpath...)
```

Comment:
I'm not sure if I understand what the previous error was - it looks like the code is doing the same thing (but is using `path` instead of `filepath`)?

Regarding `path` I think it would make sense to add a test for `CourierTemplatePath` which checks if the behaviour on windows is correct? So it is able to open files and also use the built in templates.